index.html

```diff
@@ -0,0 +1,145 @@
+                <p class="date font-face">LEARN EVERYDAY + LIFE TIME SUPPORT</p>
+                <p class="location font-face">We are fully online | Support team available 24 hours |</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="program-bg"></div>
+        <div class="p-cont">
+            <p class="program-title">Main Program</p>
+            <hr>
+        </div>
+        
+        <div class="container wrapper">
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/lecturer.png" alt="lecture image">
+                    <p class="t1">Lecture</p>
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/chat.png" alt="lecture image">
+                    <p class="t1">Forum</p>
+                </div>
+                <p class="program-disc">Have questions, drop us as chat.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/workshop.png" alt="lecture image">
+                    <p class="t1">Mentors</p>
+                </div>
+                <p class="program-disc">You are assigned a personal mentor</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/meeting.png" alt="lecture image">
+                    <p class="t1">Meetings</p>
+                </div>
+                <p class="program-disc">We have weekly meetings to address challenges we are faced.</p>
+            </div>
+        </div>
+        <div class="button">
+            <a class="start-learning" href="#">Start Learning</a>
+        </div>
+        <a class="whole-program">SEE THE WHOLE PROGRAM</a>
+    </section>
+     <!-- speaker section -->
+     <section class="container main-speakers">
+        <div class="p-cont">
+            <p class="program-title">Featured Speakers</p>
+            <hr>
+        </div>
+
+        <div id="dynamic-page" class="row"></div>
+        <div id="more" class="row grid"></div>
+      
+        <div class="read-button">
+            <button onclick="ReadMore()" class="read-more" id="toggle">MORE<i class="fa fa-angle-down fa-lg"></i></button>
```

Comment:
[Optional] I recommend that you include an arrow pointing down when the text is more and let it change to an arrow pointing upwards when the text is less. This will make the user experience better by letting them know that there's more content.

Your design:
![image](https://user-images.githubusercontent.com/20554583/197324420-fc312e88-a4ef-4997-8757-2504d8a84ebb.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/197324445-e32eda1d-cdf9-49df-b0b0-5cc152edb9ce.png)